### ISSUE TITLE
Using this tool to play audio does not support Chinese. I hope to fix this problem                       

### ISSUE THREAD

**Issue by hea7enn** [szmikler/playsound3#8]
Playing does not support Chinese. As long as the audio file has Chinese in the path or the audio file name has Chinese, it cannot be played. I hope you can fix this problem. Thank you!!

**Comment by szmikler** [comment]
Please help me to reproduce this problem.

I attempted to reproduce this on Linux. I renamed an .mp3 file to "蚊紫酱.mp3" and tried to play it:

```
>>> from playsound3 import playsound
>>> playsound("蚊紫酱.mp3")
```

It works just fine.

What system are you using?

**Comment by hea7enn** [comment]
I use Win10 system

**Comment by hea7enn** [comment]
I use the Win10 system
That is, if there is Chinese in the path where the audio file is placed, it cannot be played
For example, the path is：D:\file\音频文件\aa.wav     At this time, the audio file will not be able to be played, as long as the path contains Chinese, it will not work

**Comment by hea7enn** [comment]
I tried it, but playing this Chinese audio file doesn't work either: E:\MyVideo\Source\Example\音频.mp3
报错信息：
MCI error code: 275
MCI error code: 263
MCI error code: 263

system：Windows 10 专业版
version：22H2

**Comment by hea7enn** [comment]
> Please help me to reproduce this problem.
> 
> I attempted to reproduce this on Linux. I renamed an .mp3 file to "蚊紫酱.mp3" and tried to play it:
> 
> ```
> >>> from playsound3 import playsound
> >>> playsound("蚊紫酱.mp3")
> ```
> 
> It works just fine.
> 
> What system are you using?

Hello, has this problem been dealt with?

**Comment by szmikler** [comment]
Hey. Yes, I think I just fixed it in version 2.5.2.

I am closing the issue now but please reopen if it still doesn't work.
